XamarinCommunityToolkit.UnitTests/Converters/ListIsNullOrEmptyConverter_Tests.cs

```diff
@@ -1,10 +1,43 @@
-﻿using System;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using XamarinCommunityToolkit.Converters;
+using Xunit;
+
 namespace XamarinCommunityToolkit.UnitTests.Converters
 {
     public class ListIsNullOrEmptyConverter_Tests
     {
-        public ListIsNullOrEmptyConverter_Tests()
+        public static IEnumerable<object[]> GetData()
```

Comment:
Not that it really matters, but is this required to be in a method? Can't it just be a collection that is declared as a variable?

Reply:
Yes, you are right. Will change it in the next PR.